### PR TITLE
GithubCI,Controls.Sample.Gtk: add github CI and fix sample build

### PR DESCRIPTION
**Pull request by aarani**
### Description of Change

This commits remove the unsupported Fill property and adds
GithubCI to build the sample project.

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "Microsoft.Build.NoTargets": "3.3.0"
   },
   "sdk": {
-    "version": "6.0.400",
+    "version": "6.0.300",
```
Comment (jsuarezruiz) [inline global.json:10]: Why are we moving to the previous version?

Comment (aarani) [reply global.json:10]: Not sure, I think the reason was that GTKSharp workflow was only supported up to 6.0.300, @parhamsaremi can you elaborate?

Comment (lytico) [reply global.json:10]: > GTKSharp workflow was only supported up to 6.0.300

this should be done upstream in GtkSharp.
I opened an issue: https://github.com/GtkSharp/GtkSharp/issues/400

Comment (knocte) [reply global.json:10]: but what's wrong with going back to .300? I mean, if it works, why not support an older version?

Comment (lytico) [reply global.json:10]: anyway, it will be removed in later versions of maui:
https://github.com/dotnet/maui/pull/10492

Comment (lytico) [reply global.json:10]: workload package is here: https://github.com/orgs/GtkSharp/packages/nuget/package/GtkSharp.NET.Sdk.Gtk.Manifest-6.0.400

Comment (knocte) [reply global.json:10]: @parhamsaremi can we revert the 400->300 changes now then? ^

Comment (parhamsaremi) [reply global.json:10]: @knocte I'll work on it.

Comment (knocte) [reply global.json]: @lytico @jsuarezruiz FYI, your feedback has been addressed

**Comment by lytico** [comment]
the build is ok:

> Successfully installed workload(s) gtk.

so i think we should take that pr.
i have some changes concerning package / workload usages in the solution, but i'll make an extra PR then i'm finished.

File: src/Controls/samples/Controls.Sample.Gtk/SimpleSampleApp/SimpleSampleGtkApplication.cs [hunk 2]
```diff
@@ -25,7 +25,6 @@ Widget OnTopContainerOverride(Widget nativePage)
 		{
 			var b = new Box(Orientation.Vertical, 0)
 			{
-				Fill = true,
 				Expand = true,
```
Comment (lytico) [inline src/Controls/samples/Controls.Sample.Gtk/SimpleSampleApp/SimpleSampleGtkApplication.cs:28]: we had an api change in GktSharp:
https://github.com/GtkSharp/GtkSharp/commit/afc7ab450b90b8cdde122a99eed690e29b5a1da7
so this is needed to use the latest version

**Comment by lytico** [comment]
manifest is on nuget.org now:
https://www.nuget.org/packages/GtkSharp.NET.Sdk.Gtk.Manifest-6.0.400/

lastest version 3.24.24.95 too:
https://www.nuget.org/packages/GtkSharp

**Comment by knocte** [comment]
> manifest is on nuget.org now:

Do you mean with this that we should remove the commit titled `GithubCI: skip manifest update` from this PR?

**Comment by aarani** [comment]
@lytico @jsuarezruiz We switched the package source to nuget instead of github packages.

**Comment by aarani** [comment]
@jsuarezruiz Any comment on this PR?

File: .github/workflows/build-gtk.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+name: Build Controls.Sample.Gtk
+on: [push, pull_request]
+
+jobs:
+  build_and_test:
+    name: Build & Test
+    runs-on: ubuntu-22.04
+    env:
+      GtkSharpVersion: 3.24.24.95
+      DotnetVersion: 6.0.400
```
Comment (jsuarezruiz) [inline .github/workflows/build-gtk.yml:10]: I think we must update this after merge https://github.com/jsuarezruiz/maui-linux/pull/66

**Review by jsuarezruiz — CHANGES_REQUESTED** [review]

File: .github/workflows/build-gtk.yml (hunk 3 above)
Comment (lytico) [reply .github/workflows/build-gtk.yml:10]: this is done in [lytico/merge/main240321](https://github.com/lytico/maui/tree/lytico/merge/main240321)
with a net8-version of GtkSharp (currently -develop), see: https://github.com/lytico/maui/blob/bf56fbbdc417fc8958e5ca58711890f5db1ee19a/.github/workflows/build-gtk.yml#L9-L11

**Pull request closed without merging**